utils.js

```diff
@@ -0,0 +1,130 @@
+
+const initialHiddenClassElements = document.getElementsByClassName('initial-hidden');
+const resetButton = document.getElementById('reset-button');
+const startButton = document.getElementById('start-button');
+const userThrowImgDiv = document.getElementById('user-throw-img');
+const opponentThrowImgDiv = document.getElementById('opponent-throw-img');
+const userThrowSpan = document.getElementById('user-throw-span');
+const opponentThrowSpan = document.getElementById('opponent-throw-span');
+const userScoreSpan = document.getElementById('user-score');
+const opponentScoreSpan = document.getElementById('opponent-score');
+const tieScoreSpan = document.getElementById('tie-score');
+const resetSpanCount = document.getElementById('reset-count');
+let userWin = 0;
+let opponentWin = 0;
+let tie = 0;
+let resets = 0;
+
```

Comment:
you have some extra white space here, cleaning up double lines can make it appear cleaner :)